index.html

```diff
@@ -507,7 +512,66 @@ <h3>StatusList2021Credential</h3>
 <a href="#bitstring-encoding"></a>.
               </td>
             </tr>
-
+            <tr>
+              <td>
+                credentialStatus.ttl
+              </td>
+              <td>
+                The <code>ttl</code> indicates the "time to live" in milliseconds.
+                This property MAY be present, but if not present, implementers MUST 
+                use a value of <code>300000</code> for this property.  A verifier
+                MUST NOT use a cached <code>StatusList2021Credential</code> that was
+                cached more for more than the <code>ttl</code> duration prior to the
+                start of verification operation on a <a>verifiable credential</a>.
+                Implementations that publish the status list SHOULD align
+                any protocol-specific caching information, such as the 
+                HTTP `Cache-Control` header, with the value in this field.
```

Comment:
Why have this in addition to `validFrom` and `validUntil`?

If we have both of these, then we have these scenarios to consider:

1. Use cases where TTL and validity period are the same; `ttl` isn't useful here.
2. Use cases where TTL is longer than the validity period. Is this valid to do? Should the spec warn against it? Why would it be done?
3. Use cases where TTL is shorter than the validity period. Why do this if the VC will continue to be valid? Couldn't this set false expectations or lead to confusion?
4. Use cases where there is a TTL but no validity period. Why do this instead of setting a validity period?

Reply:
To clarify on question 3 -- I don't think this is a backwards compatible change. That's fine (we're not to CR yet), but the PR opening text says it is. A verifier that knew nothing of `ttl` and previously just checked `validFrom` and `validUntil` would do the wrong thing here -- they MUST check `ttl` as well, because `validFrom` and `validUntil` now become invalid outside of `ttl`.

To me, that seems like you're invalidating `validFrom` and `validUntil`. Perhaps `ttl` should only be used *instead of* `validFrom` and `validUntil`, which is also a breaking change, but seems more clear.

This is sort of coming at the purpose of `ttl` in reverse -- suggesting that the use case is to prevent reissuance / resigning of credential status list VCs when no VC statuses have changed. Is that the use case?

Reply:
Ah good catch - to would definitely be a change that would break things.  We should break this apart further.  Valid until deals with how often the credential needs to be reissued even if no status has changed, ttl deals with client caching behavior.  Those two things may or may not align.

Reply:
I have struggled with the problem of how long to cache these status list credentials in the past. thanks for addressing this issue so clearly.

If I were hosting this credential status list, I would not want to sign a new one for each access request.  The TTL field might require me to sign a new credential for every request.  Otherwise, I could get conflicts between TTL and valid Until.  

If we want a separate caching hint, we may want a cacheUntil with an absolute time.  Then I could generate these periodically and re-use the same document, or I could generate one for each request to spread out the time when requestors re-up.   I don't know how this would be different than changing validUntil.

Reply:
@paulfdietrich I think validUntil expiration would be what triggers the new credential creation - TTL is a directive at the verifier to retrieve a new copy of the list (newly issued or not) in case there were updates to that list.  Let me see if there is a way to improve the language to make that clear.

Reply:
I agree with Mike's breakdown: 
TTL = how long to cache the credential.
ValidUntil = the credential has expired a new one should be issued.